package.json

```diff
@@ -23,11 +23,13 @@
   "license": "BSD-3-Clause",
   "dependencies": {
     "backbone": "1.2.3",
+    "cartocolor": "^3.0.0",
```

Comment:
I feel we should make this dependency fixed. Let it open could bring some troubles in the future.